packages/fxa-react/components/AppErrorDialog/index.tsx

```diff
@@ -43,6 +51,36 @@ const AppErrorDialog = ({ errorType }: { errorType?: ErrorType }) => {
             </p>
           </Localized>
         )}
+
+        {errorType === 'invalid-session' && (
+          <>
+            <Localized id="app-invalid-session-err-heading">
+              <h2
+                className="text-grey-900 font-header text-lg font-bold mb-3"
+                data-testid="error-invalid-session"
+              >
+                Invalid Session
+              </h2>
+            </Localized>
+            <Localized id="app-invalid-session-err-message">
+              <p className="text-grey-400">
+                Please click{' '}
```

Comment:
"click here" type links are generally discouraged for accessibility, perhaps "Please <link>sign in</link> again" would work? I'm not sure we need to mention both sign out and sign in if the link will handle the sign out and redirect to sign in.

In any case, this is an improvement on the previous error, we can run the specific error message by Jeff and tweak later.

Reply:
+1 to Valerie's comment. FWIW even moving "click here" to the end of the sentence would be nicer. When you put something like, "Click here to xyz", screen reader users land on "click here" and may need to navigate forward to have the rest of the sentence read, and then navigate backwards to actually click it. I feel like "Please <sign out> and sign in again" with sign out being the link might be best?

> In any case, this is an improvement on the previous error, we can run the specific error message by Jeff and tweak later.

I'll be removing this in the UX follow up next sprint anyway.